ocaml-lsp-server/test/e2e/__tests__/textDocument-signatureHelp.ts

```diff
@@ -288,31 +337,93 @@ describe_opt("textDocument/completion", () => {
           ],
           documentation: {
             kind: "markdown",
-            value: outdent`
-              This function has a nice documentation.
-
-              It performs division of two integer numbers.
-              * * *
-              ***@param*** \`x\` dividend
-
+            value: outdent`This is an example of a docstring that demonstrates various ocamldoc syntax features.
+              
+              ### Sections and Labels
+              
+              We can create sections using 
+              
+              ### Section title
+              
+              and labels using 
+              
+              ### Section title with label
+              
+              ### Links and Cross-references
+              
+              External links: [OCaml's official website](https://ocaml.org/)
+              
+              ### Inline Formatting
+              
+              **Bold**, *Italic*, *Emphasize*, Superscript, Subscript, and \`inline code\`
+              
+              ### Text Alignment
+          
+              Centered text
+              
+              Left-aligned text
+              
+              Right-aligned text
+              
+              ### Lists
+              
+              1.Ordered list item 1
+              2.Ordered list item 2
+          
+              *Unordered list item 1
+              *Unordered list item 2
+
+              *Unordered list item 1
+              *Unordered list item 2
+              
+              ### Code Blocks
+              
+              \`\`\`
+              let square x = x * x
+              let result = square 3
+              let result = square 3
```

Comment:
@dbuenzli I'll investigate more to confirm this isn't a problem in my code, but at first sight, it looks like cmarkit is duplicating the last line of code blocks. I'll open an issue once I manage to get a proper repro, but pinging you in case this rings a bell.